README.md

```diff
@@ -64,21 +74,16 @@ From the Linux command line
 
 `jupyter lab`
 
```

Comment:
shall we add some "close your work" instructions, such as
`conda deactivate geonet-tutorial` ?

Reply:
I'm unsure about the extent to which we should be providing instructions for managing python environments. Am adding an official `conda` link to this.

I already put `ipykernel` in the `environment.yml` file so you don't need to install it there. Am removing that line.

Online I've see arguments for why you should install JupyterLab in your `base` environment and then access other environments from within JupyterLab. That is the way I do it. Have seen others say its okay to install it in each environment you might use it in. Haven't found an official pronouncement from JupyterLab or elsewhere on what you should do. So don't think we should be telling someone how they should do it. What is currently in the tutorial implies, but does not explicitly say, Jupyterlab is installed in the `geonet-tutorial` environment, but it isn't (at least not in the environment we supply). So have changed it slightly.

Have tried to specify how to launch JupyterLab from Windows, but not 100% sure I have it right.

Reply:
yes fair enough. I can't test the MacOS/Windows options, but agree that users shall probably refer to official documentation, and here we provide just a quick overview. good call again!